RetroBar/Languages/English.xaml

```diff
@@ -75,6 +75,7 @@
     <s:String x:Key="debug_logging">Enable _debug logging</s:String>
     <s:String x:Key="check_for_updates">Check for u_pdates</s:String>
     <s:String x:Key="show_exit_menu_item">Show E_xit option in right-click menu</s:String>
+    <s:String x:Key="show_end_task_button">Show End _task button in context menu</s:String>
```

Comment:
For consistency with the wording of the item above, let's use `Show End _task option in right-click menu`